monkey/infection_monkey/exploit/weblogic.py

```diff
@@ -58,6 +58,16 @@ def get_exploit_config(self):
         exploit_config['url_extensions'] = URLS
         return exploit_config
 
+    def exploit_host(self):
+        exploiters = [WebLogic20192725]
```

Comment:
There must be a better way to do this

Reply:
Where's the second exploiter?

Reply:
This is the second exploiter. It calls the new exploiter and if it fails proceeds to try itself.

Reply:
@ShayNehmad is there a better way to go trough the WebLogic exploiters? I wanted to at least move the exploiter array out of the class but got `unresolved reference` error.